beancount_n26/__init__.py

```diff
@@ -85,6 +99,20 @@ def name(self):
     def file_account(self, _):
         return self.account
 
+    def file_date(self, file_):
```

Comment:
Just to understand better, are you trying to return the most recent date from this function or the date in the last line?

Reply:
I was actually trying to return the most recent date which happened to be in the last line of my statements. I just update the method to clarify the behavior!